live-examples/js-examples/string/meta.json

```diff
@@ -1,5 +1,19 @@
 {
     "pages": {
+        "stringIterator": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/string/string-iterator.html",
+            "fileName": "string-iterator.html",
+            "title": "JavaScript Demo: String.prototype[@@iterator]()",
+            "type": "js"
+        },
+        "stringTrim": {
```

Comment:
This should be stringIteratorForeach or something (but I don't think you actually need this example).